pylint/checkers/refactoring.py

```diff
@@ -438,11 +438,17 @@ def visit_assign(self, node):
         else:
             return
 
-        self.add_message(
-            'consider-using-ternary', node=node,
-            args=(truth_value.as_string(),
-                  cond.as_string(),
-                  false_value.as_string()),)
+        if truth_value.bool_value() is False:
+            # TBD: different message? What would be a good name?
```

Comment:
I'd need an insightful input on this. 

What should be content of warning message in this case?
What should be message symbol for both those cases? Same? Different?